pack/side/tmg_encounter.json

```diff
@@ -0,0 +1,1411 @@
+    "faction_code": "seeker",
+    "health": 2,
+    "illustrator": "Tiziano Baracchi",
+    "is_unique": true,
+    "name": "Caldwell Philips",
+    "pack_code": "tmg",
+    "position": 22,
+    "quantity": 1,
+    "sanity": 3,
+    "skill_agility": 1,
+    "skill_intellect": 1,
+    "skill_willpower": 1,
+    "subname": "Enthralled by Legends",
+    "text": "You get +1 [intellect].\n[reaction] After you discover 1 or more clues, exhaust Caldwell Philips: Draw that number of cards (to a maximum of 3).\n<b>Forced</b> - When Caldwell Philips leaves play: Remove him from the game.",
+    "traits": "Leader. Guest. Miskatonic.",
+    "type_code": "asset"
+  },
+  {
+    "code": "71022b",
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 22,
+    "enemy_evade": 3,
+    "enemy_fight": 2,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "health": null,
+    "hidden": true,
+    "illustrator": "Tiziano Baracchi",
+    "is_unique": true,
+    "name": "Caldwell Philips",
+    "pack_code": "tmg",
+    "position": 22,
+    "quantity": 1,
+    "subname": "Compelled by Dreains",
+    "text": "Aloof. Hunter.\nCaldwell Philips cannot be damaged.\n<b>Forced</b> - When you end your turn at this location: Choose and discard a non-weakness card from your hand.",
+    "traits": "Humanoid. Leader. Rival. Elite.",
+    "type_code": "enemy",
+    "victory": 0
+  },
+  {
+    "code": "71023",
+    "cost": 2,
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 23,
+    "faction_code": "seeker",
+    "flavor": "\"Disturbingly lifelike, isn't it?\"",
+    "health": 2,
+    "illustrator": "Tiziano Baracchi",
+    "is_unique": true,
+    "name": "Dr. Mya Badry",
+    "pack_code": "tmg",
+    "position": 23,
+    "quantity": 1,
+    "sanity": 2,
+    "skill_agility": 2,
+    "skill_intellect": 2,
+    "slot": "Ally",
+    "subname": "Medical Examiner",
+    "text": "Your maximum hand size is increased by 2.\n[action] Exhaust Dr. Mya Badry: <b>Investigate.</b> Your base value for this investigation is equal to the number of cards in your hand.",
+    "traits": "Ally. Guest. Miskatonic.",
+    "type_code": "asset"
+  },
+  {
+    "code": "71024",
+    "cost": 2,
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 24,
+    "faction_code": "seeker",
+    "health": 3,
+    "illustrator": "Robert Laskey",
+    "is_unique": true,
+    "name": "Lucas Tetlow",
+    "pack_code": "tmg",
+    "position": 24,
+    "quantity": 1,
+    "sanity": 1,
+    "skill_combat": 2,
+    "skill_intellect": 2,
+    "slot": "Ally",
+    "subname": "Faculty Curator",
+    "text": "[reaction] After the last clue is discovered at your location, exhaust Lucas Tetlow: Search the top ? cards of your deck for an [[Item]] asset and draw it. ?If it is? a [[Relic]], gain 2 resources. If it is a [[Tome]], you ?may? play it <i>(paying its cost)</i>. If it is a [[Tool]], discover ?? at a connecting location. Shuffle your deck.",
```

Comment:
Could not make out the full card text in the spoiler images here.

Reply:
Full card text:

```
[reaction] After the last clue is discovered at your location, exhaust Lucas Tetlow: Search the top 9 cards of your deck for an [[Item]] asset and draw it. If it is a [[Relic]], gain 2 resources. If it is a [[Tome]], you may play it <i>(paying its cost)</i>. If it is a [[Tool]], discover 1 clue at a connecting location. Shuffle your deck.
```